cmd/server/cmd/server.go

```diff
@@ -20,11 +20,13 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net"
+	"net/http"
 	"os"
 	"strings"
 	"time"
 
 	bt "github.com/opentracing/basictracer-go"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
```

Comment:
can we also add _ "expvar" ? It doesn't hurt, and mostly free. Just needs the default http handler to be exposed.